### PR TITLE
Fix http broken when room has no variables

### PR DESCRIPTION
**Pull request by egalvis39**

### CODE REVIEW

File: menuflow/nodes/http_request.py [hunk 1]
```diff
@@ -130,6 +130,10 @@ async def request(self, session: ClientSession, middleware: HTTPMiddleware) -> T
             await self.room.update_menu(node_id=o_connection, state=None)
             return 500, e
 
+        # If we don't have variables for the template return
+        if not self.room._variables:
+            return response.status, await response.text()
```
Comment (bramenn) [inline menuflow/nodes/http_request.py]: Returns if it has no variables, but what if it has o_connections? As you are told to choose one of the cases. Apparently the return doesn't consider that.

**Review by bramenn — CHANGES_REQUESTED** [review]

File: menuflow/nodes/http_request.py (hunk 1 above)
Comment (egalvis39) [reply menuflow/nodes/http_request.py]: We should let that the execution continue?, we can have problems with others jinja templates during the execution

Comment (bramenn) [reply menuflow/nodes/http_request.py]: If there are variables, keep them, otherwise continue as expected.

Comment (egalvis39) [reply menuflow/nodes/http_request.py]: I did the correction

**Review by bramenn — APPROVED** [review]

**Pull request merged**